vortex-serde/src/layouts/read/layouts/chunked.rs

```diff
@@ -0,0 +1,115 @@
+use std::collections::VecDeque;
+
+use bytes::Bytes;
+use vortex_error::{vortex_err, VortexResult};
+use vortex_flatbuffers::footer;
+
+use crate::layouts::read::buffered::BufferedReader;
+use crate::layouts::read::cache::RelativeLayoutCache;
+use crate::layouts::{LayoutDeserializer, LayoutId, LayoutReader, LayoutSpec, ReadResult, Scan};
+
+#[derive(Debug)]
+pub struct ChunkedLayoutSpec;
+
+impl ChunkedLayoutSpec {
+    pub const ID: LayoutId = LayoutId(2);
```

Comment:
should pull these magic ID constants out somewhere, alongside FILE_POSTSCRIPT_SIZE, MAGIC_BYTES, INITIAL_READ_SIZE, etc.